src/ApiFunction.ts

```diff
@@ -55,17 +55,17 @@ export class ApiFunction extends Construct {
    * @param filterPattern Pattern to filter by (default: containing ERROR)
    */
   private monitor(filterPattern?: IFilterPattern) {
-    const filter = new MetricFilter(this, 'MetricFilter', {
+    const errorMetricFilter = new MetricFilter(this, 'MetricFilter', {
```

Comment:
iets duidelijkere naam bijv: error-metric-filter oid

Reply:
Dat snap ik niet helemaal, de variabele heet nu toch `errorMetricFilter`?

Reply:
Ik bedoel de resource naam, die is nu 'MetricFilter'

Reply:
Hmm ik snap wat je bedoelt, maar dit is een construct en die wordt op 4 plekken gebruikt, ik weet niet precies wat de impact is als we de resource name veranderen.

Reply:
Oke voor mij prima, maar dacht wellicht wil je het veranderen